From ab0fe092c9d0ed4e4846fe6dc6aa44151954afab Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maik=20W=C3=B6hl?= <maik.woehl@outlook.de>
Date: Tue, 27 Feb 2024 22:45:11 +0100
Subject: [PATCH 5/6] Make init.d/xpra stateless
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Update references in init.d/xpra to use stateless config directory in
/usr/share/defaults.

Signed-off-by: Maik Wöhl <maik.woehl@outlook.de>
---
 fs/etc/init.d/xpra | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fs/etc/init.d/xpra b/fs/etc/init.d/xpra
index 00b211014..d0c68333b 100755
--- a/fs/etc/init.d/xpra
+++ b/fs/etc/init.d/xpra
@@ -21,7 +21,7 @@ else
 fi
 
 # Read configuration variable file if it is present
-[ -e /etc/sysconfig/xpra ] && . /etc/sysconfig/xpra
+[ -e /usr/share/defaults/xpra/xpra.env ] && . /usr/share/defaults/xpra/xpra.env
 [ -e /etc/default/xpra ] && . /etc/default/xpra
 # Load the VERBOSE setting and other rcS variables
 [ -f /etc/default/rcS ] && . /etc/default/rcS
@@ -49,7 +49,7 @@ start() {
   echo 'Starting service…' >&2
   xpra proxy $VIRTUAL_DISPLAY \
     --bind=/run/xpra/system --auth=$AUTH \
-    --bind-tcp=0.0.0.0:$PORT --tcp-auth=$TCP_AUTH --ssl-cert=/etc/xpra/ssl-cert.pem \
+    --bind-tcp=0.0.0.0:$PORT --tcp-auth=$TCP_AUTH --ssl-cert=/usr/share/defaults/xpra/ssl-cert.pem \
     --socket-permissions=666 --debug=$DEBUG \
     --daemon=yes --log-file=$LOGFILE --log-dir=$LOGDIR --pidfile=$PIDFILE
   if [ "$?" = "0" ];then
-- 
2.44.0

